src/primitives.cc

```diff
@@ -118,15 +118,16 @@ class PrimitiveNode : public LeafNode
  * @return radius value of type Value::ValueType::NUMBER or Value::ValueType::UNDEFINED if both
  *         variables are invalid or not set.
  */
-Value PrimitiveModule::lookup_radius(const Context &ctx, const std::string &diameter_var, const std::string &radius_var) const
+Value PrimitiveModule::lookup_radius(const Context &ctx, const ModuleInstantiation *inst, const std::string &diameter_var, const std::string &radius_var) const
```

Comment:
Why are we passing a `ModuleInstantiation` here, when we usually pass a `Location` reference?

Reply:
good suggestion. This makes the indention obvious and limits accidential exposure of interfaces.

The only reason I  used ModuleInstantiation was, because I tend to copy paste the line number code.